valhalla/jawn/src/lib/handlers/LytixHandler.ts

```diff
@@ -33,8 +36,13 @@ export class LytixHandler extends AbstractLogHandler {
       modelOutput: JSON.stringify(context.processedLog.response.body) ?? "",
       metaData: context.processedLog.request.properties ?? {},
       session:
-        context.message.log.request.properties?.["Helicone-Session-ID"] ?? "",
+        context.message.log.request.properties?.["Helicone-Session-Id"] ?? "",
+      sessionPath:
+        context.message.log.request.properties?.["Helicone-Session-Path"] ?? "",
+      sessionName:
+        context.message.log.request.properties?.["Helicone-Session-Name"] ?? "",
       userIdentifier: context.message.log.request.userId,
+      logId: context.message.log.request.id ?? "",
```

Comment:
Also adding logId here, I believe this is what for feedback: https://docs.helicone.ai/features/advanced-usage/feedback

We wanted to add this so we can do evaluations on feedback (e.g. 👍 / 👎 )